src/services/processProvider.ts

```diff
@@ -60,39 +60,46 @@ export class UnixProcessProvider implements ProcessProvider {
     }
 }
 
-function getWmicValue(line: string): string {
-    const index = line.indexOf('=');
-
-    return line.substring(index + 1);
+interface WmiWin32ProcessObject {
+    readonly CommandLine: string | null;
+    readonly Name: string;
+    readonly ProcessId: number;
 }
 
 export class WindowsProcessProvider implements ProcessProvider {
     async listProcesses(filePath: string): Promise<ProcessInfo[]> {
         // WMIC lists processes by file name regardless of its full path.
         const fileName = path.basename(filePath);
 
-        const list = await Process.exec(`wmic process where "name='${fileName}' or name='${fileName}.exe'" get commandline,name,processid /format:list`);
-        
-        // Lines in the output are delimited by "<CR><CR><LF>".
-        const lines = list.stdout.split('\r\r\n');
+        const list = await Process.exec(
+            `Get-WmiObject -Query "select CommandLine, Name, ProcessId from win32_process where Name='${fileName}' or Name='${fileName}.exe'" | Select-Object -Property CommandLine, Name, ProcessId | ConvertTo-Json`,
+            {
+                shell: 'powershell.exe'
```

Comment:
This assumes that the powershell.exe is in the path. Are we fine with this assumption?

Reply:
The PowerShell docs seem to imply that PowerShell is both installed and in `PATH` by default, so I'm ok with that assumption until we find a customer that says otherwise.